Sources/PostgresMigrations/Migrations.swift

```diff
@@ -226,8 +225,8 @@ public actor DatabaseMigrations {
     /// so it is best to run this with dryRun set to true before running it without so you know what migrations
     /// it will revert.
     ///
-    /// For a migration to be removed it has to have been registered either using ``DatabaseMigrations.apply(_:)``
-    /// or ``DatabaseMigrations.register(_:)``.
+    /// For a migration to be removed it has to have been registered  either using ``DatabaseMigrations.apply(_:)``
+    /// and ``DatabaseMigrations.register(_:)``.
```

Comment:
What the purpose of this edit? It's not correct English anymore. You've changed it from "either A or B" to "either A and B"